src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -179,6 +183,36 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_codecov_path(self, source_url: str) -> Optional[str]:
```

Comment:
Move all codecov functions and constants into `src/sentry/integrations/utils/codecov.py` and import it here. It will help in the long term for re-usability and testing purposes.